### PR TITLE
Add activate/deactivate scripts for csh and fish

### LINKED ISSUE #43: activate/deactivate do not support csh and variants
Issue: When activating a conda environment in csh (or tcsh or other variants) with `source <path>/profile.d/conda.csh`, the environment variables for this shell are not set, causing any packages that depend on `PROJ_LIB`  (e.g. basemap) to fail to import.

I'm happy to provide more details about the conda environment, etc. if they are relevant but i think the problem is fairly simply that there are only scripts for `sh` and not `csh` here:
https://github.com/conda-forge/proj.4-feedstock/tree/master/recipe/scripts

and added here:
https://github.com/conda-forge/proj.4-feedstock/blob/master/recipe/build.sh#L18-L19

Presumably a corresponding set of `.csh` scripts would take care of this issue?

### PR DESCRIPTION
**Pull request by xylar**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

closes #43

### CODE REVIEW

**Comment by xylar** [comment]
I tested activating and deactivating environments with `proj4` under both `csh` and `fish` shells and the `PROJ_LIB` environment variable was set, then unset as expected.

This is my first time working with `fish` so another set of eyes on those scripts wouldn't hurt.

I didn't test activating one proj4 env and then another and going back so I'll test that now.

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
Yep, seems to work (both in `csh` and `fish`) even when going between environments, both of which have `proj4`:
```
% conda activate test_proj4
% env | grep PROJ
PROJ_LIB=/global/homes/x/xylar/miniconda2/envs/test_proj4/share/proj
% conda activate test_proj4_2
% env | grep PROJ
PROJ_LIB=/global/homes/x/xylar/miniconda2/envs/test_proj4_2/share/proj
% conda deactivate
% env | grep PROJ
PROJ_LIB=/global/homes/x/xylar/miniconda2/envs/test_proj4/share/proj
```

**Comment by xylar** [comment]
@conda-forge/proj4, please review once CI has finished.

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
@msarahan @ocefpaf @pelson @snowman2, CI now ran successfully and this is ready for review.  Thank you in advance!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -56,4 +56,5 @@ extra:
     - ocefpaf
     - pelson
     - msarahan
+    - xylar
```
Comment (ocefpaf) [inline recipe/meta.yaml:59]: :+1:

**Comment by xylar** [comment]
Thanks @ocefpaf for the clean-up and merge!  I'm going to also do a build for the last version before 6.0 so it will work with basemap.

**Pull request merged**